.github/workflows/ignore-intermittent.txt

```diff
@@ -1,3 +1,2 @@
 tests/tail/inotify-dir-recreate
 tests/misc/timeout
-tests/rm/rm1
```

Comment:
I think the intermittent was depending on how the removal was done. If it is in alphabetic order, it would not work but in random, it could